libs/shared/src/lib/survey/global-properties/others.ts

```diff
@@ -7,8 +7,11 @@ import {
   Serializer,
   matrixDropdownColumnTypes,
   settings,
+  SurveyModel,
+  PageModel,
+  surveyLocalization,
 } from 'survey-core';
-import { SurveyModel, PageModel, surveyLocalization } from 'survey-core';
+// Assuming these are your local imports
```

Comment:
@vladi2998 
just be aware of the text AI can add to files and that should be removed 👍